docker/Dockerfile-alpine

```diff
@@ -0,0 +1,14 @@
+FROM python:3.7.3-alpine
```

Comment:
Could be better if the image sticks to the Python 3.7 series instead of a specific 3.7 patch? (use `python:3.7-alpine`)